pkg/plugins/backendplugin/pluginextensionv2/rendererv2.proto

```diff
@@ -18,14 +18,28 @@ message RenderRequest {
   int32 timeout = 8;
   string timezone = 9;
   map<string, StringList> headers = 10;
-  string renderType = 11;
 }
 
 message RenderResponse {
   string error = 1;
+}
+
+message RenderCSVRequest {
+  string url = 1;
+  string filePath = 5;
+  string renderKey = 6;
+  string domain = 7;
+  int32 timeout = 8;
+  string timezone = 9;
+  map<string, StringList> headers = 10;
+}
+
+message RenderCSVResponse {
+  string error = 1;
   string fileName = 2;
 }
 
 service Renderer {
   rpc Render(RenderRequest) returns (RenderResponse);
-}
+  rpc RenderCSV(RenderCSVRequest) returns (RenderCSVResponse);
+}
```

Comment:
Nit, but can you add back the end of file newline? As you can see from the GitHub diff's warning, it's expected for text files to end with a newline.